libs/documentation/src/lib/components/sidenavigation/demos/optional/sidenavigation-optional.component.ts

```diff
@@ -0,0 +1,71 @@
+import { Component, ViewChild, AfterViewInit, ChangeDetectorRef, OnInit } from '@angular/core';
+import { BehaviorSubject } from 'rxjs';
+import { SideNavigationModel, NavigationMode, NavigationLink } from '@gsa-sam/components'
+import { ActivatedRoute } from '@angular/router';
+import { FormlyFieldConfig, FormlyFormOptions } from '@ngx-formly/core';
+import { FormGroup } from '@angular/forms';
+import { navigationConfig } from './navigate.config';
+
+@Component({
+  templateUrl: 'sidenavigation-optional.component.html'
+})
+
+export class SideNavigationOptional implements OnInit  {
+  public navigationModel: SideNavigationModel = navigationConfig;
+  form:FormGroup;
+  filterModel: any;
+  results: any;
+  public filterChange$ = new BehaviorSubject<object>(null);
+  fields: FormlyFieldConfig[] = [
+    {
+      key: 'filters',
+      templateOptions: {
+        label: 'Entity Information',
+       group: 'accordion',
+      },
+      fieldGroup: [
+        {
+          key: 'keyword',
+          type: 'input',
+          templateOptions: {
+            type: 'text',
+            label: 'Unique Entity ID',
+            tagText: 'DUNS',
+            tagClass: 'sds-tag--info-purple'
+          }
+        },
+        {
+          key: 'multiple.accordion.entity.title',
+          type: 'input',
+          templateOptions: {
+            label: 'Entity Name',
+            placeholder: 'Acme Corporation',
+            description: 'Enter the name of your entity.',
+            required: true
+          }
+        },
+        {
+          key: 'expirationDateOption',
+          type: 'radio',
+          templateOptions: {
+            label: 'Expiration Date',
+            options: [
+              { label: '30 Days', value: '30' },
+              { label: '60 Days', value: '60' },
+              { label: '90 Days', value: '90' },
+
+            ]
+          },
+        },
+      ]
+    }
+  ];
+  ngOnInit() {
```

Comment:
remove function -- results are not used anywhere

Reply:
removed